pyproject.toml

```diff
@@ -54,7 +54,7 @@ env_files = """
     .env.test
     .env
 """
-markers = ["unit"]
+markers = ["unit", "integration"]
```

Comment:
Instead of marking all these tests as integration tests using pytest, can we mark them as S3 tests instead and be more explicit? To run all the integrations tests, we would run pytest on the whole folder -- so this 'mark' isn't currently representative of what it says it's going to do